README.md

```diff
@@ -34,3 +34,44 @@ For example:
 And the same regex syntax works to run the same set:
 
     $ java -jar target/jmh-jdk-microbenchmarks-1.0-SNAPSHOT.jar .*bulk_par_lambda.*
+
+## Troubleshooting
+
+### Build of micros-javac module got stuck
+
+If you build got stuck on `[get] Getting: https://download.java.net/openjdk/jdk11/ri/openjdk-11+28_windows-x64_bin.zip` then you are probably experiencing some networking or web proxy obstacles. 
+
+Solution is to download required reference JDK from [https://download.java.net/openjdk/jdk11/ri/openjdk-11+28_windows-x64_bin.zip](https://download.java.net/openjdk/jdk11/ri/openjdk-11+28_windows-x64_bin.zip) manually and then build the project with property pointing to the local copy:
+
+    $ mvn clean install -Djavac.benchmark.openjdk.zip.download.url=file:///<your download location>/openjdk-11+28_windows-x64_bin.zip
+
+Note: Please use `openjdk-11+28_windows-x64_bin.zip` to build the project no matter what target platform is.
+
+### Execution of micros-javac benchmarks fail with java.lang.IllegalAccessError
+
+If you experience following exception during benchmarks execution:
+
+	java.lang.IllegalAccessError: superclass access check failed: class
+	org.openjdk.bench.langtools.javac.JavacBenchmark$2 (in unnamed module) cannot access class
+	com.sun.tools.javac.main.JavaCompiler
+
+It is caused by recently enabled Jigsaw enforcement and micros-javac benchmarks requirement to access several jdk.compiler module private packages. 
+
+Solution is to export required packages by adding following options to the command line:
```

Comment:
assuming this issue comes from reflective access, what about adding a single option to permit illegal access?

Reply:
I thought it was also removed however it is only  deprecated, so to generally permit illegal access is definitely better solution. Thanks!

Reply:
fixed